test-load/test.py

```diff
@@ -5,7 +5,7 @@
 from tqdm import tqdm
 from urllib.parse import quote
 
-from usi_test_cases import test_usi_list
+from metabolomics_spectrum_resolver.usi_test_cases import test_usi_list
```

Comment:
more convenient to get this variable via the package to make this test more reproducible, but I think I need to look into the github workflow to make sure the package actually gets installed at some point